libs/shared/log/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@fxa/shared/log",
```

Comment:
Probably doesn't matter, but vscode is showing a warning for this package name. It says, `String does not match the pattern of "^(?:@(?:[a-z0-9-*~][a-z0-9-*._~]*)?/)?[a-z0-9-~][a-z0-9-._~]*$".`